tests/baselines/reference/switchStatementsWithMultipleDefaults.errors.txt

```diff
@@ -26,17 +22,15 @@ tests/cases/compiler/switchStatementsWithMultipleDefaults.ts(27,22): error TS110
             switch (x * x) {
                 default:    // No issues.
                 default:    // Error; second 'default' clause.
-                ~~~~~~~~
-!!! error TS1113: A 'default' clause cannot appear more than once in a 'switch' statement.

```

Comment:
the other errors are gone because there is a parse error which prevents semantic checks?

Reply:
yep